app/lib/frontend/templates/views/pkg/admin_page.dart

```diff
@@ -0,0 +1,159 @@
+        d.markdown(
+            '**Upgrading to verified publishers is an irreversible operation.** '
+            'Packages can be transferred between publishers, but they can\'t be converted back to legacy uploader ownership.'),
+      ],
+      if (pkgHasPublisher)
+        d.markdown(
+            'You can transfer your package to a **different publisher** if you’re also a member of the publisher.'),
+      if (userPublishers.isNotEmpty) ...[
+        material.dropdown(
+          id: '-admin-set-publisher-input',
+          label: 'Select a publisher',
+          options: [
+            if (!pkgHasPublisher)
+              d.option(value: '', text: '', disabled: true, selected: true),
+            ...userPublishers.map(
+              (p) => d.option(
+                  value: p, text: p, selected: p == package.publisherId),
+            ),
+          ],
+        ),
+        d.p(
+          child: material.button(
+            id: '-admin-set-publisher-button',
+            classes: ['pub-button-danger'],
+            raised: true,
+            label: 'Transfer to publisher',
+          ),
+        ),
+      ],
+      if (userPublishers.isEmpty)
+        d.markdown('You have no verified publisher. '
+            'Use the [create publisher](${urls.createPublisherUrl()}) page to create one.'),
+      if (!pkgHasPublisher) ...[
+        d.h3(text: 'Uploaders'),
+        material.dataTable<User>(
+          ariaLabel: 'Uploaders of package',
+          columns: [
+            material.DataTableColumn<User>(
+              headerContent: d.text('Email'),
+              headerClasses: ['email-header'],
+              renderCell: (u) => d.text(u.email!),
+            ),
+            material.DataTableColumn<User>(
+              headerContent: d.text(''),
+              headerClasses: ['icons-header'],
+              renderCell: (u) => d.a(
+                classes: ['-pub-remove-uploader-button'],
+                title: 'Remove uploader',
+                attributes: {'data-email': u.email!},
+                text: '×',
+              ),
+            ),
+          ],
+          entries: uploaderUsers,
+        ),
+        d.p(
+          child: material.button(
+            id: '-pkg-admin-invite-uploader-button',
+            label: 'Invite uploader',
+            raised: true,
+          ),
+        ),
+        d.div(
+          id: '-pkg-admin-invite-uploader-content',
+          classes: ['modal-content-hidden'],
+          children: [
+            d.p(
+                text: 'You can invite new uploader to this package. '
+                    'Once new uploaders accept the invitation, they have full administrative rights, '
+                    'with the following abilities:'),
+            d.ul(children: [
+              d.li(text: 'Transfer this package to a publisher'),
+              d.li(text: 'Upload new versions of this package'),
+              d.li(text: 'Invite and remove uploaders of this package'),
+            ]),
```

Comment:
nit. Consider using markdown..

Reply:
Let's refactor that next time this part needs an update.